lib/matplotlib/tests/test_image.py

```diff
@@ -1593,3 +1593,39 @@ def test_resample_dtypes(dtype, ndim):
     axes_image = ax.imshow(data)
     # Before fix the following raises ValueError for some dtypes.
     axes_image.make_image(None)[0]
+
+
+@pytest.mark.parametrize('intp_stage', ('data', 'rgba'))
+@check_figures_equal()
+def test_interpolation_stage_rgba_respects_alpha_param(fig_test, fig_ref, intp_stage):
+    axs_tst = fig_test.subplots(2, 3)
+    axs_ref = fig_ref.subplots(2, 3)
+    scl_a = 0.5  # scalar alpha
+    arr_a = np.random.rand(3, 3)  # array alpha
```

Comment:
Use a telling variable name instead of a cryptic one, which is explained in a comment 😃. 

To have less magic numbers and keep it more readable, I propose to variables for the number of points.

```suggestion
    ny, nx = 3, 3
    scalar_alpha = 0.5
    array_alpha = np.random.rand(ny, nx)
```